cmd/virtual-workspaces/command/cmd.go

```diff
@@ -95,7 +98,7 @@ func Run(ctx context.Context, o *options.Options) error {
 	nonIdentityConfig.Host = u.String()
 
 	// resolve identities for system APIBindings
-	nonIdentityKcpClusterClient, err := kcpclient.NewClusterForConfig(nonIdentityConfig) // can only used for wildcard requests of apis.kcp.dev
+	nonIdentityKcpClusterClient, err := kcpclient.NewForConfig(kcpclienthelper.SetCluster(rest.CopyConfig(nonIdentityConfig), tenancyv1alpha1.RootCluster)) // can only used for wildcard requests of apis.kcp.dev
```

Comment:
Was the root cluster implicit before?

Reply:
It was being scoped later in the controller. Modified this to scope it here, since there is no other use of unscoped client .